.github/workflows/preprod-deployment.yml

```diff
@@ -107,14 +107,13 @@ jobs:
 
     steps:
       - name: Check out repository code
-        uses: actions/checkout@master
+        uses: actions/checkout@v4
       
       - name: Download the VCS Agent binary
         uses: actions/download-artifact@v3
         with:
           name: vcs-agent-binary
           path: ./build
-          retention-days: 1
```

Comment:
I recommend the GitHub Actions vs code extension, it pointed this out 🙂 this parameter doesn't exist